python/lsst/daf/butler/remote_butler/_remote_butler.py

```diff
@@ -179,6 +211,57 @@ def getURI(
         # Docstring inherited.
         raise NotImplementedError()
 
+    def get_dataset_type(self, name: str) -> DatasetType:
+        # In future implementation this should directly access the cache
+        # and only go to the server if the dataset type is not known.
+        path = f"dataset_type/{name}"
+        response = self._client.get(self._get_url(path))
+        response.raise_for_status()
```

Comment:
Will this raise some HttpError instead of `MissingDatasetTypeError` when dataset type is not know?